### DISCUSSION TITLE
Reference/documentation for SITT and LITT

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#773] [Questions & Problems]
I'm looking for a reference or documentation regarding the "small intestinal transit time factor" and the "large intestinal transit time factor" parameters. I don't understand the equations for both factors as well as how the linear equation was parameterized.

**Comment by JanSchlender** [comment]
Hi moPellowe,
Have you had a look at the two publications on the establishement of the current GI-tract in PK-Sim? 
https://pubmed.ncbi.nlm.nih.gov/21993815/ 
https://pubmed.ncbi.nlm.nih.gov/22125236/ 
Best,
Jan

**Comment by PavelBal** [comment]
I did, and I didn't get far - so @moPellowe if you do understand the equations after reading these two publications, I would be happy to discuss it with you.

**Comment by Yuri05** [comment]
Intestinal transit rates were fitted to match experimental intestinal transit data 
(s. figure 1 of the article https://pubmed.ncbi.nlm.nih.gov/21993815/)

Using those factors is more for historical reasons; small/large intestinal transit time can be used directly
s. Open-Systems-Pharmacology/PK-Sim/Issues/1541

**Comment by Yuri05** [comment]
@katrincoboeken @AndreDlm FYI

**Comment by moPellowe** [comment]
Thank you for the references and the explanations. I will give them a read and let you know if I have any additional questions.

**Comment by StephanSchaller** [comment]
@moPellowe , have you figured out how the SITT factor is derived? (how P1 and P2 were fitted) and the transit times for each gut segment?

**Comment by moPellowe** [comment]
@StephanSchaller and @PavelBal , actually no, I still don't understand how the SITT factor is derived (i.e. P1 and P2). I skimmed the two articles that were given above by @JanSchlender , and I can see how the intestinal segment transit rates (before scaling?) could be derived from Figure 1A of the first article https://pubmed.ncbi.nlm.nih.gov/21993815/ as @Yuri05 mentioned, but I still don't understand where the SITT/LITT factor equations come from. Does anyone have any insight into those equations?
